docs/basics/UsageWithReact.md

```diff
@@ -271,111 +354,37 @@ By itself, it’s not very interesting. Let’s connect it to Redux!
 
 ## Connecting to Redux
 
-We need to make two changes to connect our `App` component to Redux and make it dispatch actions and read state from the Redux store.
+We need to make two changes to connect our `TodoApp` component to Redux and make it dispatch actions and read state from the Redux store.
 
 First, we need to import `Provider` from [`react-redux`](http://github.com/gaearon/react-redux), which we installed earlier, and **wrap the root component in `<Provider>`** before rendering.
 
 #### `index.js`
 
 ```js
-import React from 'react'
-import { render } from 'react-dom'
-import { createStore } from 'redux'
-import { Provider } from 'react-redux'
-import App from './containers/App'
-import todoApp from './reducers'
+import "babel-core/polyfill";
+import React from "react";
+import { render } from "react-dom";
+import { Provider } from "react-redux";
+import { todoStore } from "./store";
+import TodoApp from "./components/TodoApp";
 
-let store = createStore(todoApp)
-
-let rootElement = document.getElementById('root')
 render(
-  <Provider store={store}>
-    <App />
+  <Provider store={todoStore}>
+    <TodoApp />
   </Provider>,
-  rootElement
-)
+  document.getElementById("app")
+);
 ```
 
+That’s it! Our todo app now functions correctly.
```

Comment:
But we said "we need to make two changes". The second change follows after "that's it" and this is confusing because the text is old. Should it be removed? I'd probably want to move and edit it before defining container components because we never actually get to explain `mapStateToProps` and `connect()`. With this PR, we just jump into defining them without preparing the reader, which is a problem.